src/node/blockstorage.cpp

```diff
@@ -1206,7 +1206,7 @@ void ImportBlocks(ChainstateManager& chainman, std::span<const fs::path> import_
 {
     ImportingNow imp{chainman.m_blockman.m_importing};
 
-    // -reindex
+    // Empty datadir or -reindex
```

Comment:
This comment change not correct and should just be dropped: `m_blockfiles_indexed` is `true` (its default value) in case of an empty datadir, `false` in case of a reindex.

Reply:
Ok, then I'm confused about what causes case (1):  https://github.com/bitcoin/bitcoin/pull/31346#discussion_r1855741271

Reply:
With an empty datadir, we don't have the genesis connected at this point. But we don't enter the reindex if clause [here](https://github.com/bitcoin/bitcoin/blob/2eccb8bc5e2245e42e05b8df3d9aa264b5f21a9a/src/node/blockstorage.cpp#L1210C16-L1210C17) (the comment only refers to that), we just skip this and the loadblock block and go to the [ActivateBestChain](https://github.com/bitcoin/bitcoin/blob/2eccb8bc5e2245e42e05b8df3d9aa264b5f21a9a/src/node/blockstorage.cpp#L1261) call below.